docs/extensions/ResponseMetadata.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Response Metadata
+category: extensions
+order: 8
+---
+
+# Response Metadata
+
+This extension allows to limit the amount of control information in responses, adapted from [OData JSON Format Version 4.01 - 3.1](http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ControllingtheAmountofControlInforma).
+
+The metadata can be applied to the $resultMetadata parameter of a request to influence how much control information will be included in the response.
+
+Sample request:
+
+```
+GET /v1.1/Things(1)?$resultMetadata=off
+```
+
+## metadata=full
+
+The full list of control information that may appear in a metadata=full response is as follows:
+
+- count: the total count of a collection of entities, if requested;
+- navigationLink: the link used to retrieve the values of a navigation property;
+- nextLink: the next link of a collection with partial results;
+- selfLink: URL of an entity.
+
+## metadata=minimal
+
+metadata=minimal removes all control information listed in metadata=full section, except:
+
+- count: if requested;
+- nextLink: the next link of a collection with partial results.
+
+The created entity URLs are also removed in responses, either in location header or in content.
+
+## metadata=off
```

Comment:
OData uses `none` instead of `off`. Do you have a specific reason to change that?
It's probably more consistent to stay closer to OData.

Reply:
resultMetadata=off is not the same as OData metadata=none since from OData spec:

> The metadata=none format parameter indicates that the service SHOULD omit control information other than nextLink and count. 

nextLink is removed in resultMetadata=off

Should we make that explicit in the doc?

In fact in SensorThings case, metadata=minimal and metadata=none as defined in OData would be the same.

Reply:
Yeah, clearly stating that would be good.
Also adding the OData `none` one would be good. Especially if it's just an alias for minimal.
One of my goals for the 2.0 (develop-entityEvolution branch) is having the option to follow OData more closely, including the $metadata page.

Reply:
I added resultMetadata=none and also restricted the POST response changes to resultMetadata=off to increase OData compliance.